src/core_plugins/kibana/public/dashboard/panel/panel_utils.js

```diff
@@ -21,6 +25,31 @@ export class PanelUtils {
     return panel;
   }
 
+  // 6.3 changed the panel dimensions to allow finer control over sizing
+  // 1) decrease column height from 100 to 20.
+  // 2) increase rows from 12 to 48
+  // Need to scale pre 6.3 panels so they maintain the same layout
+  static convertPanelDataPre_6_3(panel) { // eslint-disable-line camelcase
+    panel.gridData.w = panel.gridData.w * PANEL_WIDTH_SCALE_FACTOR;
```

Comment:
FYI, I'm working on a migrations feature, which hopefully will eliminate the need for this kind of code to be done at read-time. https://github.com/elastic/kibana/issues/15100

Just noting, so we might be able to go back through and remove this kind of thing at some point, hopefully!